README.md

```diff
@@ -1,4 +1,7 @@
 # Codewind ODO extension
+[![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
+[![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
+
 Extension to Codewind providing support for ODO projects: https://codewind.dev
 
```

Comment:
For license badge:
Codewind repo uses this one:
[![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
Codewind appsody repo uses this one:
[![License](https://img.shields.io/badge/license-Eclipse-brightgreen.svg)](https://www.eclipse.org/legal/epl-2.0/)
Not sure which one is the right one, but I think we should make it consistent for all Codewind subprojects/repos

If we decide to use this one [![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/), then this PR looks good to me

Reply:
Well, appsody repo should be updated with the same one and a PR was opened already. :) https://github.com/eclipse/codewind-appsody-extension/pull/56

Reply:
@josephca Thanks, approved this PR.

Reply:
BTW, please add a build badge when you're ready to add Jenkinsfile. Thanks,

Reply:
@josephca Yes, will do that with the Jenkins build PR.